api/lib/infrastructure/repositories/schooling-registration-repository.js

```diff
@@ -55,11 +56,24 @@ module.exports = {
     const trx = await Bookshelf.knex.transaction();
 
     try {
+      const nationalStudentIdsFromFile = schoolingRegistrationDatas.map((schoolingRegistrationData) => schoolingRegistrationData.nationalStudentId);
+      const students = await studentRepository.findReconciledStudentsByNationalStudentId(nationalStudentIdsFromFile);
 
       const schoolingRegistrations = _.map(schoolingRegistrationDatas, (schoolingRegistrationData) => new SchoolingRegistration({ ...schoolingRegistrationData, organizationId }));
       const schoolingRegistrationsFromOrganization = await this.findByOrganizationId({ organizationId });
```

Comment:
```suggestion
      const currentSchoolingRegistrations = await this.findByOrganizationId({ organizationId });
```
Y'a peut être mieux que pending, mais currentlyImported c'est un peu long
Vous avez utilisé `nationalStudentIdsFromFile`, pourquoi pas `schoolingRegistrationsFromFile` ?